cleo/commands/completions_command.py

```diff
@@ -410,7 +407,7 @@ def _zsh_describe(self, value: str, description: str | None = None) -> str:
             description = re.sub(
                 r'(["\'#&;`|*?~<>^()\[\]{}$\\\x0A\xFF])', r"\\\1", description
             )
-            value += ":{}".format(subprocess.list2cmdline([description]).strip('"'))
+            value += f""":{subprocess.list2cmdline([description]).strip('"')}"""
```

Comment:
i personally have a hard time reading this, maybe drop the f-string?
```suggestion
            value += ":" + subprocess.list2cmdline([description]).strip('"')
```